diff --git a/include/gmm/gmm_precond_ilu.h b/include/gmm/gmm_precond_ilu.h
index 25d322d..10db4fd 100644
--- a/include/gmm/gmm_precond_ilu.h
+++ b/include/gmm/gmm_precond_ilu.h
@@ -207,8 +207,8 @@ namespace gmm {
       }
     }
 
-    L = tm_type(&(L_val[0]), &(L_ind[0]), &(L_ptr[0]), n, mat_ncols(A));
-    U = tm_type(&(U_val[0]), &(U_ind[0]), &(U_ptr[0]), n, mat_ncols(A));
+    L = tm_type(L_val.data(), L_ind.data(), L_ptr.data(), n, mat_ncols(A));
+    U = tm_type(U_val.data(), U_ind.data(), U_ptr.data(), n, mat_ncols(A));
   }
   
   template <typename Matrix>
